schedule/yast/yast2_ncurses/yast2_ncurses_gnome.yaml

```diff
@@ -34,7 +34,7 @@ conditional_schedule:
         - console/yast2_lan_hostname/dhcp_yes
         - console/yast2_lan_hostname/dhcp_yes_eth
         - console/yast2_settings
-        - console/yast2_dns_server
+        - yast2_gui/yast2_dns_server
```

Comment:
This 'yast2_gui' line looks a bit strange in the sequence of 'console' tests. I wonder if we can have a less "confusing" naming scheme for libyui tests that in fact would cover both GUI and console UI.

Reply:
Yes, this test is for ncurse test, it shouldn't be in yast2_gui folder, but I haven't found a suitable position for it, create a new folder for it, such as 'yast2_ncurse'?